lib/vagrant-lxc/provider.rb

```diff
@@ -36,7 +36,7 @@ def sudo_wrapper
 
       def ensure_lxc_installed!
         begin
-          sudo_wrapper.run("/usr/bin/which", "lxc-create")
+          sudo_wrapper.run("/usr/bin/env", "which", "lxc-create")
```

Comment:
Can we prepend `"/usr/bin/env"` to every path in the sudo_wrapper instead to fix this everywhere? There are some more similar cases in the networking code where tools like `ip` and `brctl` are called.